src/Microsoft.Health.Dicom.Blob/Utilities/StoreConfigurationAware.cs

```diff
@@ -0,0 +1,23 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Blob.Utilities
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    internal abstract class StoreConfigurationAware : IStoreConfigurationAware
```

Comment:
What about just having a constants file with the 4 things we need in it:
```
        internal const string MetadataConfigurationSectionName = "DicomWeb:MetadataStore";
        internal const string MetadataContainerName = "dicomMetadata";
        internal const string BlobConfigurationSectionName = "DicomWeb:DicomStore";
        internal const string BlobContainerName = "dicomBlob";
```

I think the only place this becomes a bit trickier is when registering the health checks, but I believe that can be fixed by adding a name as a parameter and using `AddTypeActivatedCheck` instead of `AddCheck`

Reply:
We could do that - one thing to note is that this is a preparation for adding a new blob type for workitems that will be in a separate container.

Reply:
> What about just having a constants file with the 4 things we need in it:
> 
> ```
>         internal const string MetadataConfigurationSectionName = "DicomWeb:MetadataStore";
>         internal const string MetadataContainerName = "dicomMetadata";
>         internal const string BlobConfigurationSectionName = "DicomWeb:DicomStore";
>         internal const string BlobContainerName = "dicomBlob";
> ```
> 
> I think the only place this becomes a bit trickier is when registering the health checks, but I believe that can be fixed by adding a name as a parameter and using `AddTypeActivatedCheck` instead of `AddCheck`

"AddTypeActivatedCheck" does take params. But it requires us passing in the resolved instances for other parameters injected in that consumer.